### PR TITLE
[SIE-142] implement preSurvey

### PR DESCRIPTION
**Pull request by alecherryy**

### CODE REVIEW

File: src/firebase/api/qualtrics.js [hunk 1]
```diff
@@ -61,25 +61,26 @@ export const getPreSurvey = async (uid, experimentId, surveyHandler) => {
...
-            data: preSurvey,
+            // data: preSurvey,
           };
           surveyHandler(res);
         } else {
           const res = {
             status: StatusCodes.NOT_FOUND,
             message: message.NOT_FOUND,
-            data: null,
+            // data: null,
```
Comment (cmutuc25) [inline src/firebase/api/qualtrics.js:83]: should we just delete these comments?

**Review by cmutuc25 — APPROVED** [review]

File: src/firebase/api/qualtrics.js (hunk 1 above)
Comment (cmutuc25) [reply src/firebase/api/qualtrics.js:83]: there's quite a few of them in qualtrics.js

**Comment by alecherryy** [comment]
@cmutuc25 I didn't delete those lines in Qualtrics because they had just been added and wanted to check-in with the team that I could in fact delete it. If my comments make sense, I'll delete them.

**Comment by alecherryy** [comment]
@cmutuc25 and @fernandowinfield I added the post-surveys and updated the AddExperiment component.

**Review by fernandowinfield — APPROVED** [review]
Looks good! 

The only thing I would like to remind us is that, for the final demo, we should probably create two more sample surveys (pre survey and post survey) so that we can show that the 'Next' button in steps 4 and 6 is disabled properly until the participant completes the survey in each respective step.

File: src/firebase/api/qualtrics.js [hunk 2]
```diff
@@ -61,25 +61,26 @@ export const getPreSurvey = async (uid, experimentId, surveyHandler) => {
...
-          };
+        if (data) {
+        // if (data && data.age) {
+          // const preSurvey = {
+          //   age: res.age,
+          //   gender: res.gender,
+          //   genderImportance: res.genderImportance,
+          //   ethnic: res.ethnic,
+          //   selfEsteem: res.selfEsteem,
+          // };
```
[the remark this answers is not in the document]
Comment (cmutuc25) [reply src/firebase/api/qualtrics.js]: I think it's fine we get rid of it then

**Review by cmutuc25 — APPROVED** [review]
This has merge issues now but otherwise it's good to go

**Pull request merged**